cve-list.html

```diff
@@ -9,6 +9,44 @@ <h1>Apache Kafka Security Vulnerabilities</h1>
 
 This page lists all security vulnerabilities fixed in released versions of Apache Kafka.
 
+      <h2 id="CVE-2023-34455"><a href="https://nvd.nist.gov/vuln/detail/CVE-2023-34455">CVE-2023-34455</a> Clients using Snappy compression may cause OutOfMemoryError on brokers</h2>
```

Comment:
What I see in the doc output is `OUTOFMEMORYERROR`, which is hard to read. Could we put them in `out of memory error`?

Reply:
updated in latest commit